astroplan/tests/test_core.py

```diff
@@ -648,10 +850,86 @@ def print_pyephem_solar_transit_noon():
     print(map(pyephem_time_to_datetime_str, [next_transit, next_antitransit,
                                              prev_transit, prev_antitransit]))
 
+def test_vega_sirius_transit_seattle():
+    """
+    Check that time of transit of Vega for an observer in Seattle is
+    consistent with PyEphem results (for no atmosphere/pressure=0)
+    """
+    lat = '47d36m34.92s'
+    lon = '122d19m59.16s'
+    elevation = 0.0 * u.m
+    pressure = 0 * u.bar
+    location = EarthLocation.from_geodetic(lon, lat, elevation)
+    time = Time('1990-01-01 12:00:00')
+    vega = SkyCoord(279.23473479*u.degree, 38.78368896*u.degree)
+    sirius = SkyCoord(101.28715533*u.degree, -16.71611586*u.degree)
+
+    obs = Observer(location=location, pressure=pressure)
+    astroplan_vega_transit = obs.calc_meridian_transit(time, vega,
+                                                       which='next').datetime
+    astroplan_sirius_transit = obs.calc_meridian_transit(time, sirius,
+                                                         which='next').datetime
+
+    astroplan_vector_transit = obs.calc_meridian_transit(time, [vega, sirius],
+                                                         which='next').datetime
+
+    # Run print_pyephem_vega_sirius_transit() to compute analogous
+    # result from PyEphem:
+    pyephem_vega_transit = datetime.datetime(1990, 1, 2, 3, 41, 9, 244067)
+    pyephem_sirius_transit = datetime.datetime(1990, 1, 1, 15, 51, 15, 135167)
+
+    # Typical difference in this example between PyEphem and astroplan
+    # with an atmosphere is <2 min
+    threshold_minutes = 8
+    assert (abs(pyephem_vega_transit - astroplan_vega_transit) <
+            datetime.timedelta(minutes=threshold_minutes))
+    assert (abs(pyephem_sirius_transit - astroplan_sirius_transit) <
+            datetime.timedelta(minutes=threshold_minutes))
+
+    # Now check vectorized solutions against scalar:
+    assert (astroplan_vector_transit[0] == astroplan_vega_transit)
+    assert (astroplan_vector_transit[1] == astroplan_sirius_transit)
+
+def print_pyephem_vega_sirius_transit():
+    """
+    To run:
+
+    python -c 'from astroplan.tests.test_core import print_pyephem_vega_sirius_transit as f; f()'
+    """
+    lat = '47:36:34.92'
+    lon = '122:19:59.16'
+    elevation = 0.0 * u.m
+    pressure = 0
+    time = Time('1990-01-01 12:00:00')
+    vega_coords = SkyCoord(279.23473479*u.degree, 38.78368896*u.degree)
+    sirius_coords = SkyCoord(101.28715533*u.degree, -16.71611586*u.degree)
+
+    import ephem
+    obs = ephem.Observer()
+    obs.lat = lat
+    obs.lon = lon
+    obs.elevation = elevation
+    obs.date = time.datetime
+    obs.pressure = pressure
+    vega = ephem.FixedBody()
+    vega._ra = ephem.degrees(np.radians(vega_coords.ra.value))
```

Comment:
Can this be simplified?
I didn't look closely, but doesn't `vega_coords.ra.deg` work?